MontyHallKata/Program.cs

```diff
@@ -1,5 +1,50 @@
-﻿// See https://aka.ms/new-console-template for more information
+﻿using MontyHallKata.Controllers;
+using MontyHallKata.Models.Randomizer;
+using MontyHallKata.Views.InputOutput;
 
-using System;
+namespace MontyHallKata
+{
+    public static class Program
+    {
+        private const int Simulation = 1;
+        private const int PlayThrough = 2;
+        private const int Quit = 0;
+        public static void Main()
+        {
+            var randomizer = new DefaultRandomizer();
+            var console = new InputOutputConsole();
+            
+            console.PrintOutput(InputOutputMessages.InitialChoicePrompt);
+            var consoleInput =  console.GetIntInput();
 
-Console.WriteLine("Hello, World!");
\ No newline at end of file
+            switch (consoleInput)
+            { 
+                case Simulation:
+                    var simulation = new SimulationGenerator(randomizer);
+            
+                    console.PrintOutput(InputOutputMessages.NumberOfSimulationsPrompt);
+                    var numberOfSimulations = console.GetIntInput();
+                    
+                    simulation.Simulate(numberOfSimulations, "stay");
+                    var stayWinningPercentage = simulation.GetWinningPercentage();
+                    simulation.Simulate(numberOfSimulations, "switch");
```

Comment:
Same as comment above re magic strings. Could be good to store them in a variable somewhere so that they can be changed easily if needed.